### ISSUE TITLE
How to use IContentRegistration.Children() in Dinamico

### ISSUE THREAD

**Issue by laurachan** [n2cms/n2cms#654]
Is this method the Dinamico equivalent of the [EditableChildren] attribute? I tried to use it as follows, but whenever I tried to add a new Accordion part I would just get the following error: "Cannot add any parts due to zone/user/type restrictions". 

Here are the Razor views; commenting out the RestrictParents, RestrictChildren and AllowedZones calls didn't make any difference.

Accordion.cshtml:
```C#
@model Dinamico.Models.ContentPart

@{
    Content.Define(re =>
    {
        re.Title = "Accordion";
        re.RestrictChildren("AccordionPanel");
        re.Children("AccordionPanels");
    });
}

<div id="accordion-@Model.ID" class="panel-group acc-v1">
    <div class="panel panel-default">
        @Html.Zone("AccordionPanels")
    </div>
</div>
```

AccordionPanel.cshtml
```C#
@model Dinamico.Models.ContentPart

@{
    Content.Define(re =>
    {
        re.Title = "Accordion Panel";
        re.RestrictParents("Accordion");
        re.AllowedZones("AccordionPanels");

        re.Text("PanelHeading", "Heading").Required();
        re.FreeText("PanelContent", "Content");
    });
}
<!-- markup omitted -->
```

**Comment by laurachan** [comment]
I looked through the N2CMS code for `EditableRegistrationExtensions.Children(this IContentRegistration registration, string zoneName)` and I see it passes the zone name to an overload of RegisterEditable<T> that seems to expect a part title and name. Could that be the reason why the Children method results in that error message?

**Comment by libardo** [comment]
There's a workaround:
		ie.Children("AccordionPanels").Configure(eca => eca.ZoneName = "AccordionPanels");

**Comment by laurachan** [comment]
Thanks for your help! May I ask if commit 42519c924d3d3a2b4816524890a99b018dbcfdf7 was included in the N2CMS 2.7.5 nuget packages that were released earlier today? If not, that's alright, I'll use the workaround.

**Comment by DejanMilicic** [comment]
I do not think so, but @libardo can correct me if I am wrong
